docs/src/main/asciidoc/_galleon/Galleon_layers.adoc

```diff
@@ -0,0 +1,863 @@
+[[wildfly_defined_galleon_layers]]
+=== WildFly Layers
+
+In the tables below we provide basic information about all of the layers WildFly provides.
+
+If the capabilities provided by a layer A require capabilities provided by another layer B, then layer A will _depend on_ layer B.
+If you ask for layer A, then Galleon will automatically provide B as well. In some cases A's dependency on B can be _optional_; that
+is A typically comes with B, but can function without it. In this case if you ask for A by default Galleon will provide B as well,
+but you can tell Galleon to _exclude_ B. The tables below detail the various dependency relationships.
+
+Some layers are logical _alternatives_ to other layers. If two layers are alternatives to each other they both provide the same general
+capabilities, but with different implementation characteristics. For example a number of layers provide the capability to cache different
+types of objects. These layers typically come in pairs of alternatives, where one alternative provides local caching, while the other provides
+distributed caching.  If a layer you want has an optional dependency on a layer that has an alternative, you can exclude that dependency
+and instead specify the alternative. If a layer has an alternative the Description column in the tables below will identify it.
+
+[NOTE]
+====
+If the elytron layer is present, security will be handled by the elytron subsystem.
+The undertow subsystem is configured with an _other_ _application-security-domain_ that references the Elytron _ApplicationDomain_ security domain.
+====
+
+[[wildfly-galleon-layers-servers]]
+==== Server Galleon layers
+
+A single Galleon layer can provide a relatively small set of capabilities, but most users will want to start with a broader set
+of capabilities without having to spell out all the details. To help with this WildFly provides a few foundational layers
+all of which provide typical core WildFly capabilities like the logging subsystem and a secure remote management interface.
+
+You don't have to base your WildFly installation on one of these basic server layers, but doing so may be more convenient.
+
+[cols="1,2,1a"]
+|===
+|Name |Description |Dependencies
+
+|[[gal.core-server]]core-server
+|A typical manageable server core.
```

Comment:
For someone coming to WildFly relatively new, does core-server really provide them anything useful?

Reply:
I will let @bstansberry answer there.

Reply:
No, but this is general documentation, and it fits in the category this table addresses.

This could use a sentence saying more what it might be used for.  "This could serve as a base for a more specialized WildFly that doesn't need the capabilities provided by the other server layers."

Then it could be moved to last in the table (which is where Jean-Francois originally had it.). I moved it up because that's a logical flow; the servers in the table expand on each others' feature set. But I think it's fine to not care about that and de-emphasize this on.

Reply:
Implemented wha tyou suggested, moved it back to the end + sentence.